debian/mariadb-plugin-mroonga.prerm

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+# Install Mroonga
+# No user or password parameter is required with new MariaDB that
+# has unix socket authentication support by default.
```

Comment:
Right, comment is not updated after move from Debian.org, but contents has `--defaults-file=/etc/mysql/debian.cnf`